.eslintrc.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  env: {
+    commonjs: true,
+    es2021: true,
+  },
+  extends: [
+    'airbnb-base',
+  ],
+  parserOptions: {
+    ecmaVersion: 12,
+  },
+  rules: {
+    'no-restricted-syntax': 'off',
+    'prefer-template': 'off',
+    'eslint-disable strict': 'off',
+    'eslint-env': 'jest'
```

Comment:
are you sure this rule is right? I can't find any reference to it. I always did it as described here https://stackoverflow.com/a/57328996/32093